Dockerfile

```diff
@@ -1,18 +1,16 @@
-FROM golang:1.10.4 AS build
-WORKDIR /go/src/github.com/lyft/ratelimit
+FROM golang:1.14 AS build
+WORKDIR /ratelimit
+
+ENV GOPROXY=https://proxy.golang.org
```

Comment:
sorry, not terribly familiar with go modules. Why is this needed in docker but not in the makefile for local builds?

Reply:
It's a mirror that can be used to speed up module downloads. We use it in Contour.